docs/source/quicktour.mdx

```diff
@@ -14,186 +14,188 @@ specific language governing permissions and limitations under the License.
+* Automatic speech recognition (ASR): transcribe audio data into text.
+
+<Tip>
+For more details about the [`pipeline`] and associated tasks, refer to the documentation [here](./main_classes/pipelines)
+</Tip>
 
-- Sentiment analysis: is a text positive or negative?
-- Text generation (in English): provide a prompt and the model will generate what follows.
-- Name entity recognition (NER): in an input sentence, label each word with the entity it represents (person, place, etc.)
-- Question answering: provide the model with some context and a question, extract the answer from the context.
-- Filling masked text: given a text with masked words (e.g., replaced by `[MASK]`), fill the blanks.
-- Summarization: generate a summary of a long text.
-- Translation: translate a text in another language.
-- Feature extraction: return a tensor representation of the text.
+### Using the pipeline
 
-Let's see how this work for sentiment analysis (the other tasks are all covered in the [task summary](task_summary)):
+In the following example, we will look at how to use the [`pipeline`] for sentiment analysis.
 
-Install the following dependencies (if not already installed):
+Install the following dependencies if you haven't already:
 
 ```bash
 pip install torch
 ===PT-TF-SPLIT===
 pip install tensorflow
 ```
 
+Import [`pipeline`] and specify the task you want to complete:
+
 ```py
 >>> from transformers import pipeline
 
 >>> classifier = pipeline("sentiment-analysis")
 ```
 
-When typing this command for the first time, a pretrained model and its tokenizer are downloaded and cached. We will
-look at both later on, but as an introduction the tokenizer's job is to preprocess the text for the model, which is
-then responsible for making predictions. The pipeline groups all of that together, and post-process the predictions to
-make them readable. For instance:
-
+The pipeline downloads and caches a default [pretrained model](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) and tokenizer for sentiment analysis. Now you can use the `classifier` on your target text:
 
 ```py
->>> classifier("We are very happy to show you the 🤗 Transformers library.")
-[{'label': 'POSITIVE', 'score': 0.9998}]
+>>> classifier("We are very happy to show you the 🤗 Transformers library."")
+[{"label": "POSITIVE", "score": 0.9998}]
 ```
 
-That's encouraging! You can use it on a list of sentences, which will be preprocessed then fed to the model, returning
-a list of dictionaries like this one:
+For more than one sentence, pass a list of sentences to the [`pipeline`] which returns a list of dictionaries:
 
 ```py
 >>> results = classifier(["We are very happy to show you the 🤗 Transformers library.", "We hope you don't hate it."])
 >>> for result in results:
-...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
+...     print(f"label: {result["label"]}, with score: {round(result["score"], 4)}")
```

Comment:
I think this change is breaking the code quality script, could you revert it? You shouldn't use `"` inside an f-string which uses `"` to define the string.